### PR TITLE
Hw16 test

### PR DESCRIPTION
**Pull request by r0var**

### CODE REVIEW

File: HW16/3args-2.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+FIRST=$1
+SECOND=$2
+THIRD=$3
+
+RE='^(-?([0-9]{1,6}|10{6})|one|two|three)$'
+
+function checkString {
+  if ! [ $# -eq 0 ]; then
+    if [[ $1 =~ $RE ]]; then
```
Comment (mavrik90) [inline HW16/3args-2.sh:11]: What about range -1000000 to 1000000 ?

Comment (r0var) [reply HW16/3args-2.sh:11]: `-?([0-9]{1,6}|10{6})`
Minus is optional ( (any digit 1-6 (0-999999) or 1 and six zeros (1000000) )
This regex expression gives a range -1000000 to 1000000

Comment (andreivaloshyn) [reply HW16/3args-2.sh:11]: Ah, too smart for me ;)

Comment (mavrik90) [reply HW16/3args-2.sh:11]: Ah, too smart for me ;)
Well done!

**Review by mavrik90 — APPROVED** [review]
LGTM

**Pull request merged**